CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 3.7.1 (2019-07-02)
+
+- Fixed `skyux build` command to ignore SPA's `fixtures` directories. [#136](https://github.com/blackbaud/skyux-sdk-builder/pull/136)
```

Comment:
Fixed the `skyux build` command to ignore a SPA's `fixtures` directories.

Reply:
Done.